mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go.erb

```diff
@@ -19,7 +22,7 @@ func TestAccAppEngineFlexibleAppVersion_update(t *testing.T) {
 
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
-		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
```

Comment:
nit: You don't need to use this for handwritten tests, it enforces in code that examples have `provider = google-beta` when they're displayed in docs and oics

non-blocking though.

Reply:
Ah thats good to know, thanks!

Reply:
Oh, but if you're removing it you *also* need to remove `provider = google-beta` from everywhere because the only TPG variant is the current provider aliased as `google` in the default factory.